Marlin/src/gcode/host/M115.cpp

```diff
@@ -245,6 +245,18 @@ void GcodeSuite::M115() {
     // CONFIG_EXPORT
     cap_line(F("CONFIG_EXPORT"), ENABLED(CONFIGURATION_EMBEDDING));
 
+    // FSR_PROBING
+    cap_line(F("FSR_PROBING"), ENABLED(HAS_FSR) || ENABLED(HAS_Z_MIN_ADC));
+
+    // SINGLE_NOZZLE
+    cap_line(F("SINGLE_NOZZLE"), ENABLED(SINGLENOZZLE));
+
+    // HEATED_BED
+    cap_line(F("HEATED_BED"), ENABLED(HAS_HEATED_BED));
+
+    // DELTA
+    cap_line(F("DELTA"), ENABLED(DELTA));
```

Comment:
We already have KINIMETATICS info, so why does this need to be added here?

https://github.com/MarlinFirmware/Marlin/blob/9a1c993719b30dabfb2ab62101ba249dffa459c7/Marlin/src/gcode/host/M115.cpp#L66-L77

Reply:
You are correct - we don't need this.